examples/hello-world/public/index.html

```diff
@@ -0,0 +1,31 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
```

Comment:
Both favicon and the comment below can be completely removed, they are irrelevant to this project.